### DISCUSSION TITLE
Use the quick exit key to also go from the beatmap results to the beatmap list

### DISCUSSION THREAD

**Discussion by Wheisss** [ppy/osu#27919] [Ideas] [answered]
It's quite niche, but as a left-handed player which uses the numpad to play and constantly make use of the quick retry and quick exit key buttons, more often than not i find myself trying to go back to the beatmap list by pressing the quick exit button and failing miserably (haha). Since the quick retry already work in the beatmap results to play again, i think it would make sense and be quite convenient for the quick exit to work on the results; maybe it could even be used as an alternative ESC button while on the game menus too.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/27922
